scripts/js/lib/markdownImages.test.ts

```diff
@@ -38,6 +38,11 @@ test("Test the finding of invalid images", async () => {
     `;
   const images = await collectInvalidImageErrors(markdown);
   const correct_images = new Set([
+    "Convert 'img1.png' to AVIF. You can use the command `magick public/images/img1.png public/images/img1.avif`. If ImageMagick isn't preinstalled, you can get it from https://imagemagick.org/script/download.php. Then delete the old file and update the markdown to point to the new file.",
+    "Convert 'img2.png' to AVIF. You can use the command `magick public/images/img2.png public/images/img2.avif`. If ImageMagick isn't preinstalled, you can get it from https://imagemagick.org/script/download.php. Then delete the old file and update the markdown to point to the new file.",
+    "Convert 'invalid_img1.png' to AVIF. You can use the command `magick public/images/invalid_img1.png public/images/invalid_img1.avif`. If ImageMagick isn't preinstalled, you can get it from https://imagemagick.org/script/download.php. Then delete the old file and update the markdown to point to the new file.",
+    "Convert 'invalid_img2.png' to AVIF. You can use the command `magick public/images/invalid_img2.png public/images/invalid_img2.avif`. If ImageMagick isn't preinstalled, you can get it from https://imagemagick.org/script/download.php. Then delete the old file and update the markdown to point to the new file.",
+    "Convert 'invalid_img3.png' to AVIF. You can use the command `magick public/images/invalid_img3.png public/images/invalid_img3.avif`. If ImageMagick isn't preinstalled, you can get it from https://imagemagick.org/script/download.php. Then delete the old file and update the markdown to point to the new file.",
```

Comment:
Can you please convert 1-2 of the test inputs to avif so that our test validates we don't have false positives?